taipy/gui_core/_context.py

```diff
@@ -446,11 +446,8 @@ def edit_entity(self, state: State, id: str, payload: t.Dict[str, str]):
                         tasks = data.get("task_ids")
                         if entity._get_name() != name:
                             if scenario := t.cast(Scenario, core_get(entity.owner_id)):
-                                with scenario:
-                                    scenario.add_sequence(name, tasks)
-                                    scenario.remove_sequence(entity._get_name())
-                                    entity = scenario.sequences[name]
-                        else:
+                                scenario.rename_sequence(entity._get_name(), name)
```

Comment:
I'm worried not to see any exception catch here...

Reply:
https://vscode.dev/github/Avaiga/taipy/blob/feature/%23292-CRUD-sequence/taipy/gui_core/_context.py#L425